frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -133,6 +133,43 @@ export default class FirebaseInteractor {
     });
   }
 
+  /**
+   * Creates a new assessment with all of the words from a given intervention.
+   */
+  async createAssessmentFromIntervention(setId: string) {
+    let intervention = await this.db
+      .collection("interventions")
+      .doc(setId)
+      .get();
+    let interventionData = intervention.data();
+    if (interventionData == null) {
+      throw new Error(`Intervention with id ${setId} does not exist`);
+    }
+
+    let wordList = interventionData.wordList;
```

Comment:
While this looks super great, I think that you are supposed to have the new assessment's words be the ones that were incorrect from the previous **assessment**, not intervention. Since we technically don't have a previous assessment right now, you can probably get one hardcoded assessment and do it from there.

Reply:
Would this mean that after EACH of the 8 interventions created for an assessment, there is the same assessment (with only incorrect words) recreated?

Reply:
Yes (like the assessment gets smaller as they get words correct)

Reply:
Updated to hardcode a previous assessment and add the incorrect words to the new intervention!